### PR TITLE
Add documentation for single engagement

### PR DESCRIPTION
**Pull request by SteamRiC**
https://github.com/salemove/application_services/pull/837
https://github.com/salemove/web_services/pull/646
https://github.com/salemove/acceptance_tests/pull/335

### CODE REVIEW

**Comment by p2tris** [comment]
+1

File: apiary.apib [hunk 1]
```diff
@@ -632,6 +632,59 @@ The grammar for a NOT filter is as follows:
 
 # Group Engagements
 
+## Single Engagement [/engagements/{engagement_id}]
+
+### Single engagement [GET]
+Fetches a single engagement by id.
+The manager needs to have access to the site that the engagement belongs to.
```
Comment (willmore) [inline apiary.apib]: "The manager needs to" =>  "The manager must"

Comment (willmore) [inline apiary.apib]: Dangling participle...
"The manager needs to have access to the site that the engagement belongs to."
=>
"The manager must have access to the site to which the engagement belongs."

**Comment by willmore** [comment]
attention @indrekj

File: apiary.apib [hunk 2]
```diff
@@ -632,6 +632,59 @@ The grammar for a NOT filter is as follows:
...
+            "href": "https://api.salemove.com/visitors/1"
+          },
+          "chat_transcript":
+          {
+            "href": "https://api.salemove.com/engagements/1/chat_trascript"
+          },
+          "operators":
+          [
+            {
+              'href' => "http://api.salemove.com/operators/1"
```
Comment (indrekj) [inline apiary.apib]: json doesn't have `=>`

**Comment by SteamRiC** [comment]
Updated

**Comment by willmore** [comment]
+1

File: apiary.apib [hunk 3]
```diff
@@ -690,10 +743,10 @@ The collection is paginated and sorted by id (descending).
...
                   {
-                    'href' => "http://api.salemove.com/operators/1"
+                    "href": "http://api.salemove.com/operators/1"
                   },
                   {
-                    'href' => "http://api.salemove.com/operators/2"
+                    "href": "http://api.salemove.com/operators/2"
                   },
                 ]
               },
```
Comment (stenver) [inline apiary.apib]: Maybe we should add engagement href to here and dump the rest, so the output would be:
```
         {
             "last_page": "https://api.salemove.com/engagements?page=2",
             "next_page": "https://api.salemove.com/engagements?page=2",
             "engagements": [
               {
                 "id": 1,
                 "href": "http://api.salemove.com/engagements/1"
                },
               {
                 "id": 2,
                 "href": "http://api.salemove.com/engagements/2"
                },
              ]
```
What say you @indrekj ?

**Comment by willmore** [comment]
!squash

**Pull request merged**